src/s390_ecc.c

```diff
@@ -468,7 +468,8 @@ unsigned int make_ecdsa_verify_parmblock(char *pb,
  * makes an ECDSA key structure at given struct and returns its length.
  */
 unsigned int make_ecdsa_private_key_token(unsigned char *kb,
-		const ICA_EC_KEY *privkey, uint8_t curve_type)
+		const ICA_EC_KEY *privkey, unsigned char *X, unsigned char *Y,
```

Comment:
can X and Y parameters be ualified be const? they seem to be read only. also applies to make_ecdsa_sign_request

Reply:
Yes, will do.